charts/airflow/README.md

```diff
@@ -1130,6 +1130,30 @@ externalDatabase:
   properties: ""
 ```
 
+<h3>Option 3 - MsSQL</h3>
+
```

Comment:
We should add a warning that MsSQL support is EXPERIMENTAL as of airflow `2.2.3`.

See the ["requirements" section](https://github.com/apache/airflow#requirements) of the airflow readme.